lib/div-alg.gi

```diff
@@ -1962,7 +1962,11 @@ for i in [1..k] do
         r1 := hrs[j][2];
         d := Gcd(m,r1-1);
         md := m/d;
-        a := Int(ZmodnZObj(e[i,j]/d,md)*ZmodnZObj((r1-1)/d,md)^-1 );
+        if j in x then
+          a := Int(ZmodnZObj(e[i,j]/d,md)*ZmodnZObj((r1-1)/d,md)^-1 );
+        else
+          a := 0;
```

Comment:
The case of `a=0` is the only changed line not covered by the tests

Reply:
It make sense because you need a very special not very small group to cover that part. I think that the following will cover that: 
gap> QG:=GroupRing(Rationals,SmallGroup(672,622));                  
<algebra-with-one over Rationals, with 7 generators>
gap> wd:=WedderburnDecompositionInfo(QG);;
gap> A:=wd[28];
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 14 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]
gap> KillingCocycle(A);
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 14 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]

Reply:
@angeldelriomateos This example depends on some randomness:
```
########> Diff in /home/travis/build/gap-packages/wedderga/gaproot/pkg/wedderg\
a/tst/div-alg.tst:46
# Input is:
A:=wd[28];
# Expected output:
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 14 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]
# But found:
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 42 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]
########
########> Diff in /home/travis/build/gap-packages/wedderga/gaproot/pkg/wedderg\
a/tst/div-alg.tst:49
# Input is:
KillingCocycle(A);
# Expected output:
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 14 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]
# But found:
[ 1, NF(7,[ 1, 6 ]), 84, [ [ 2, 13, 42 ], [ 2, 29, 63 ], [ 2, 43, 0 ] ], 
  [ [ 42, 0 ], [ 63 ] ] ]
########
```
Is it acceptable to check that `wd[28]=KillingCocycle(wd[28])` instead?